sources/us/ny/statewide.json

```diff
@@ -14,6 +14,14 @@
     "conform": {
         "type": "geojson",
         "street": "CompleteStreetName",
-        "number": "AddressNumber"
+        "number": ["PrefixAddressNumber", "AddressNumber", "SuffixAddressNumber"],
+        "unit": "SubAddress",
+        "city": "ZipName",
+        "region": "State",
+        "postcode": "ZipCode",
```

Comment:
Bah, humbug. Good catch.

Same rights restrictions [apparently also apply to ZipName](http://gis.ny.gov/gisdata/inventories/details.cfm?DSID=921).

Switching that over as well.